src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -136,32 +137,34 @@ def import_config(cmd,
         src_kvs = __read_kv_from_app_service(
             cmd, appservice_account=appservice_account, prefix_to_add=prefix, content_type=content_type)
 
-    # if customer needs preview & confirmation
-    if not yes:
+    if strict or not yes:
```

Comment:
I'm concerned that the if-else branches in this method have gotten too complicated. We might miss some edge cases here and there keeping track of these conditions. Is it possible to simplify it?

Reply:
I agree with you. Keeping up with various options we support has led to complicated if-else branches.
I would ideally keep only branchng logic here, and use helper methods for separate options. (e.g. what we did for kvset)
But it is a larger change and needs to be scoped out of this PR.

I am open to any suggestions you have to simplify this logic in the meantime.

Reply:
Okay, we can refactor this complicated logic in future.